cmd/runtimes/deepspeed/Dockerfile

```diff
@@ -0,0 +1,42 @@
+FROM nvidia/cuda:12.4.1-devel-ubuntu22.04
+
+# Disable interactive dialog from apt.
+ENV DEBIAN_FRONTEND noninteractive
+
+# Install libraries required for OpenMPI to work.
+RUN apt-get update && apt install -y --no-install-recommends \
+    cmake g++ gcc \
+    wget vim \
+    openssh-client openssh-server libcap2-bin \
+    libopenmpi-dev openmpi-bin
```

Comment:
I downgraded OpenMPI version since 5.0 is not required for DeepSpeed to work.
That should speedup the image building time.